ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -388,13 +397,65 @@ TorchCore::processIValue (torch::jit::IValue value, GstTensorMemory *output, uns
   output_tensor = output_tensor.contiguous ();
 
   /* validate output tensor once */
-  if (configured < outputTensorMeta.num_tensors && validateOutputTensor (output_tensor, idx)) {
+  if (configured < outputTensorMeta.num_tensors
+&& validateOutputTensor (output_tensor, idx)) {
     ml_loge ("Output Tensor Information is not valid");
     return -1;
   }
 
   /** @todo avoid this memcpy */
-  std::memcpy (output->data, output_tensor.data_ptr (), output_tensor.nbytes ());
+  std::memcpy (output[idx].data, output_tensor.data_ptr (), output_tensor.nbytes ());
+  return 0;
+}
+
+/**
+ * @brief	serialize and process the output from the invoke
+ * @param[in] value IValue containing the output in tensor form
+ * @param[out] output Output tensor memory
+ * @param[inout] idx index of output
+ * @return 0 if OK. non-zero if error.
+ *         -2 if output tensor validation fails.
+ *         -3 if output is of unsupported format.
+ */
+int
+TorchCore::serializeOutput (
+    const torch::jit::IValue &value, GstTensorMemory *output, unsigned int *idx)
```

Comment:
Indentation looks strange here, too.

Reply:
I am not sure what is the desired indentation here as the clang-format recommends this indentation. Please let me know the desired format and I will update at other locations in this file too.